backend/app/assets/javascripts/spree/backend/format_money.js

```diff
@@ -0,0 +1,10 @@
+(function() {
+  var defaultLocale = document.documentElement.lang;
+
+  Spree.formatMoney = function(amount, currency, options) {
```

Comment:
This is not a temporary fix. The frontend and backend have different javascript, different requirements, and different browser support.